app/models/user.rb

```diff
@@ -0,0 +1,33 @@
+require 'bcrypt'
+
+class User < ActiveRecord::Base
+  has_many :teams
+  include BCrypt
+
+  validates :username, :presence => {:message => "Type in username you must"}
+  validates :username, :uniqueness => {:message => "Please stop trying to steal other people's usernames."}
+
+  def password
+    @password ||= Password.new(password_hash)
+  end
+
+  def password=(new_password)
+    @password = Password.create(new_password)
+    self.password_hash = @password
+  end
+
+  def create
+    @user = User.new(params[:user])
+    @user.password = params[:password]
+    @user.save!
+  end
+
+  def login
```

Comment:
login is the responsibility of Sinatra not a model. This method needs to be replaced with a sinatra helper.